dist/Makefile.tmpl

```diff
@@ -180,18 +178,24 @@ CTYPES_CMX    =  $(filter-out $(BLACKLIST_CMX),$(CTYPES_DEPS))
 CTYPES_CMX    += lib/Lib_RandomBuffer_System_stubs.cmx lib/Lib_RandomBuffer_System_bindings.cmx
 CTYPES_ML     =  $(patsubst %.cmx,%.ml,$(CTYPES_CMX))
 CTYPES_CMI    =  $(patsubst %.cmx,%.cmi,$(CTYPES_CMX))
+CTYPES_CMO    = $(patsubst %.cmx,%.cmo,$(CTYPES_CMX))
+
+
+ocamlevercrypt.cma: $(ALL_BINDINGS) $(CTYPES_CMO) $(ALL_C_STUBS) $(CTYPES_CMX) libevercrypt.$(SO)
+	ocamlmklib -o ocamlevercrypt $(CTYPES_CMO) -L. -levercrypt
 
 ocamlevercrypt.cmxa: $(ALL_BINDINGS) $(ALL_ML_STUBS) $(ALL_C_STUBS) libevercrypt.$(SO)
-	ocamlfind mklib -o ocamlevercrypt $(CTYPES_CMX) $(ALL_C_STUBS) -ccopt -L. -cclib -levercrypt
+	ocamlmklib -o ocamlevercrypt $(CTYPES_CMX) -L. -levercrypt
```

Comment:
why not prefix with `ocamlfind`?

Reply:
I followed the example in the docs, but the result seems to be the same